pkg/cfg/cfgmgr.go

```diff
@@ -238,8 +274,25 @@ func (c *CfgMgr) CreateDB(serviceName, userName string) error {
     }
     _, _, _, err = c.ConsulKV.Txn(ops, nil)
     if err != nil {
-        zap.L().Error("Can't write db config to Consul, please retry later")
+        zap.L().Error("Can't write db config to Consul", zap.Error(err))
```

Comment:
```suggestion
        zap.L().Error("Can't write db config to Consul", err)
```
Does this not work?

Reply:
no. it gave `pkg/cfg/cfgmgr.go:279:23: cannot use err (type error) as type zapcore.Field in argument to zap.L().Error` this error while compiling.